pkg/ipam/node.go

```diff
@@ -202,6 +202,12 @@ func (n *Node) getMinAllocate() int {
 	return n.resource.Spec.ENI.MinAllocate
 }
 
+// getMaxAllocate returns the maximum-allocation setting of an AWS node
+//
```

Comment:
Nit: Can we remove this commented, but otherwise empty line?

Reply:
sounds good, done! 👍